webpack.development.js

```diff
@@ -60,7 +60,7 @@ const browserConfig = (isLegacy = false) => ({
           new ForkTsCheckerWebpackPlugin({
             async: true,
             typescript: {
-              mode: 'write-references', // for better babel-loader perf.
+              mode: 'write-references',
```

Comment:
@coldlink Could you clarify if `'write-references'` remains because it is improving perf of `swc-loader` as well or if there is another reason to keep it?

Reply:
Good spot, after digging into the plugin documentation, it seems that setting this property wasn't actually doing anything now or previously, so I'll remove it.

> Use readonly if you don't want to write anything on the disk, write-dts to write only .d.ts files, write-tsbuildinfo to write only .tsbuildinfo files, write-references to write both .js and .d.ts files of project references (last 2 modes requires build: true).

That documentation suggests that because we're not using the `build: true` property in this plugin, the `mode` property was ignored anyway.

https://github.com/TypeStrong/fork-ts-checker-webpack-plugin#typescript-options